Features/PrefixCommand.cs

```diff
@@ -0,0 +1,36 @@
+using DSharpPlus.Commands;
+using DSharpPlus.Entities;
+using DSharpPlus.Commands.ContextChecks;
+using ArtcordAdminBot.Features.Helpers;
+
+
+namespace ArtcordAdminBot.Features
+{
+    public class PrefixCommand
+    {
+        private readonly IDatabaseService _databaseService;
+
+        public PrefixCommand(IDatabaseService databaseService)
+        {
+            _databaseService = databaseService;
+        }
+
+        [Command("setprefix")]
+        [System.ComponentModel.Description("Set the bot's prefix")]
+        //[RequirePermissions(DiscordPermissions.Administrator)]
+        public async Task SetPrefixAsync(CommandContext ctx, string newPrefix)
+        {
+            if (string.IsNullOrWhiteSpace(newPrefix))
+            {
+                await ctx.RespondAsync("Prefix cannot be empty.");
+                return;
+            }
+
+            await _databaseService.SetPrefixAsync(ctx.Guild.Id, newPrefix);
+
+            await ctx.RespondAsync(
+                MessageHelpers.GenericEmbed("",$"Prefix successfully updated to `{newPrefix}`.", "#00ffff")
```

Comment:
I think we have a `GenericSuccessEmbed`? If yes, use it instead, otherwise we should create it, but we can do that in another commit

```suggestion
                MessageHelpers.GenericSuccessEmbed(title:"Prefix updated", description:$"Prefix successfully updated to `{newPrefix}`.")
```